packit_service/worker/events/event.py

```diff
@@ -194,6 +193,21 @@ def db_project_event(self) -> Optional[AbstractProjectEventDbType]:
                     project_url=self.project_url,
                     commit_hash=self.commit_sha,
                 )
+            elif self.event_type in {
+                "NewHotnessUpdateEvent",
+            }:
+                if self.project:
+                    namespace = self.project.namespace
+                    repo_name = self.project.repo
+                else:
+                    namespace = repo_name = None
```

Comment:
> Not sure if setting namespace and repo name to None won't break anything in dashboard for example. Would it be better to try to somehow extract the values from project_url?

I think we could utilise `RepoUrl` that is already used in `NewHotnessEvent` implementation to have those values, WDYT?

Reply:
Thanks, done.